symengine/lib/symengine_wrapper.pyx

```diff
@@ -37,6 +37,12 @@ cdef c2py(RCP[const symengine.Basic] o):
     elif (symengine.is_a_Pow(deref(o))):
         r = Basic.__new__(Pow)
     elif (symengine.is_a_Integer(deref(o))):
+        if (deref(symengine.rcp_static_cast_Integer(o)).is_zero()):
+            return S.Zero
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_one()):
+            return S.One
+        elif (deref(symengine.rcp_static_cast_Integer(o)).is_minus_one()):
+            return S.NegativeOne
         r = Number.__new__(Integer)
     elif (symengine.is_a_Rational(deref(o))):
         r = Number.__new__(Rational)
```

Comment:
How can I check for numerator and denominator equals 1 and 2 respectively for half here?